express-get-json/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const grades = [
+  {
+    12: {
+      id: 12,
+      name: "Tim Berners-Lee",
+      course: "HTML",
+      score: 95,
+    },
+    47: {
+      id: 47,
+      name: "Brendan Eich",
+      course: "JavaScript",
+      score: 100,
+    },
+    273: {
+      id: 273,
+      name: "Forbes Lindsay",
+      course: "JavaScript",
+      score: 92,
+    },
+  },
+];
+
+const app = express();
+
+app.get("/api/grades", (req, res) => {
+  res.send(grades);
```

Comment:
once you change `grades` to an object of objects, you will have to loop through it, push the objects into a new array, and send that array to the client.